src/Sarif.Viewer.VisualStudio/Sarif/Run.extensions.cs

```diff
@@ -136,5 +136,48 @@ public static bool HasErrorLevelToolExecutionNotifications(this Run run)
 
             return false;
         }
+
+        /// <summary>
```

Comment:
>/// [](start = 8, length = 3)

Would these helpers be better off in the SDK?

Reply:
Agreed, nice helpers. Create an SDK issue?

---
In reply to: [525564884](https://github.com/microsoft/sarif-visualstudio-extension/pull/276#discussion_r525564884) [](ancestors = 525564884)

Reply:
[#2157](https://github.com/microsoft/sarif-sdk/issues/2157): Migrate suppressed and absent issue helpers from VS extension to SDK

---
In reply to: [527881875](https://github.com/microsoft/sarif-visualstudio-extension/pull/276#discussion_r527881875) [](ancestors = 527881875,525564884)